tests/unit/client/sdk/v1/metadata_properties/test_api.py

```diff
@@ -0,0 +1,50 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from uuid import uuid4
+
+import httpx
+import pytest
+from argilla.client.sdk.v1.metadata_properties.api import update_metadata_property
+
+
+@pytest.mark.asyncio
+class TestSuiteMetadataPropertiesSDK:
+    async def test_update_metadata_property(self, mock_httpx_client: httpx.Client) -> None:
```

Comment:
I think we dont need asyncio support for these test suite
```suggestion
class TestSuiteMetadataPropertiesSDK:
    def test_update_metadata_property(self, mock_httpx_client: httpx.Client) -> None:
```

Reply:
True, I'll undo that!